lib/omnibus-flapjack/publish.rb

```diff
@@ -198,6 +199,28 @@ def add_to_rpm_repo(pkg, component = 'experimental')
           Mixlib::ShellOut.new('createrepo . 2>&1', :live_stream => $stdout).run_command.error!
         end
       end
+
+      def add_to_packagecloud(pkg, component='experimental')
+        begin
+          credentials = Packagecloud::Credentials.new(ENV['packagecloud_user'], ENV['packagecloud_token'])
```

Comment:
Probably also skip the call to add_to_packagecloud if the credentials aren't found in the environment, up in the Rakefile level